mediorum/Dockerfile.dev

```diff
@@ -17,10 +74,20 @@ RUN cmake -DCMAKE_INSTALL_PREFIX=/usr/local -DBUILD_TESTING=OFF -S . -B build &&
     cmake --build build --parallel $(nproc) && \
     cmake --install build
 
+# Clone and build Essentia
+ENV PKG_CONFIG_PATH="/usr/local/lib/pkgconfig:/usr/lib/pkgconfig"
+ENV LD_LIBRARY_PATH="/usr/local/lib:/usr/lib"
+RUN git clone https://github.com/MTG/essentia.git /essentia
+WORKDIR /essentia
+RUN git checkout v2.1_beta5 && \
```

Comment:
Is there a reason for using `v2.1_beta5`?

Reply:
yeah it's the "current stable version recommended to install" according to the docs https://essentia.upf.edu/installing.html#compiling-essentia